convert/gitlab/yaml/pipeline.go

```diff
@@ -14,16 +14,24 @@
 
 package yaml
 
+import (
+	"reflect"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
 type (
 	// Pipeline defines a gitlab pipeline.
 	Pipeline struct {
-		Default   *Default             `yaml:"default,omitempty"`
-		Include   []*Include           `yaml:"include,omitempty"`
-		Image     *Image               `yaml:"image,omitempty"`
-		Jobs      map[string]*Job      `yaml:",inline"`
-		Stages    []string             `yaml:"stages,omitempty"`
-		Variables map[string]*Variable `yaml:"variables,omitempty"`
-		Workflow  *Workflow            `yaml:"workflow,omitempty"`
+		Default      *Default             `yaml:"default,omitempty"`
+		Include      []*Include           `yaml:"include,omitempty"`
+		Image        *Image               `yaml:"image,omitempty"`
+		Jobs         map[string]*Job      `yaml:"jobs,omitempty"`
```

Comment:
this needs to remain `inline` for correct marshaling back to yaml

Reply:
No, the UnmarshalYAML function in pipeline.go takes care of it.